var/spack/test_repos/spack_repo/builtin_mock/packages/py_extension1/package.py

```diff
@@ -3,12 +3,12 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 import os
 
-from spack_repo.builtin_mock.build_systems.python import PythonPackage
+from spack_repo.builtin_mock.build_systems.python import PythonExtension
 
 from spack.package import *
 
 
-class PyExtension1(PythonPackage):
+class PyExtension1(PythonExtension):
```

Comment:
```suggestion
class PyExtension1(Package, PythonExtension):
```
I'm pretty sure if you do this (and import `from spack_repo.builtin_mock.build_system.generic import Package`) that will fix the test failure